tests/Directory.Build.props

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
you first remove the Directory.Build.props from tests, and now you recover it?

Reply:
Yes 😢, This is to `disable` `warnings` and set `coverlet.msbuild` package. So i can clean up `.github` workflows.